src/source/Sctp/Sctp.c

```diff
@@ -63,15 +65,33 @@ STATUS initSctpSession()
     // Disable Explicit Congestion Notification
     usrsctp_sysctl_set_sctp_ecn_enable(0);
 
+    pSctpSessionControl = MEMALLOC(SIZEOF(SctpSessionControl));
+    pSctpSessionControl->lock = MUTEX_CREATE(FALSE);
+    CHK_STATUS(hashTableCreate(&pSctpSessionControl->activeSctpSessions));
```

Comment:
Use the parameterized version of the hash table. The default will create 2 * 10K buckets